taxonomies/packaging_shapes.result.txt

```diff
@@ -388,3 +424,9 @@ fr:Muselet, Muselet et plaque
 en:wrapper, wrapping
 fr:enveloppage
 
+nl:Folder
```

Comment:
Would you have English translations for those?

Reply:
added, would be brochures or leaflets